leshan-core/src/main/java/org/eclipse/leshan/core/node/codec/text/LwM2mNodeTextEncoder.java

```diff
@@ -100,6 +101,10 @@ public void visit(LwM2mResource resource) {
                 ObjectLink objlnk = (ObjectLink) val;
                 strValue = String.valueOf(objlnk.getObjectId() + ":" + objlnk.getObjectInstanceId());
                 break;
+            case OPAQUE:
+                byte[] binaryValue = ((byte[])val);
```

Comment:
formatting issue : no space between `(byte[])` and `val`

I can fix this myself, but If you plan to contribute often there is [an eclipse formatter file](https://github.com/eclipse/leshan/wiki/Code-&-design-guidelines#configure-your-ide).
I don't know if `Visual Studio Code` is able to handle it ?

If you don't, I can easily fix this before to integrate your commit, just let me know !

Reply:
No problem, I will try without Eclipse. If you see other problems  just let me know.